jetpack-beta.php

```diff
@@ -508,6 +516,13 @@ static function get_jetpack_plugin_pretty_version( $is_dev_version = false ) {
 			return 'Latest Stable';
 		}
 
+		if ( 'tags' === $section ) {
+			return sprintf(
+				'Official <a href="https://plugins.trac.wordpress.org/browser/jetpack/tags/%s" target="_blank">tag</a> downloaded from wp.org svn',
```

Comment:
```suggestion
				__( 'Public beta (<a href="https://plugins.trac.wordpress.org/browser/jetpack/tags/%1$s" target="_blank" rel="">available on WordPress.org</a>', 'jetpack-beta' ),
```

Reply:
Most of the tags are not beta, but the other suggestions look good!

Reply:
That's a good point, I'll replace with `release`.